telemetry/opting-out-of-telemetry.adoc

```diff
@@ -0,0 +1,21 @@
+[id="opting-out-of-telemetry"]
+= Opting out of Telemetry
+include::modules/common-attributes.adoc[]
+:context: telemetry
+
+toc::[]
+
+You might need to opt out of Telemetry for your cluster.
+
+[NOTE]
```

Comment:
This note needs to be conditionalized to only display in OSD. Double-check with Vikram.